components/signIn/signIn.css

```diff
@@ -0,0 +1,96 @@
+@import url('../../style.css');
+
+.login-container{
+    display: flex;
+    flex-direction: column;
+    justify-content:center;
+    align-items: center;
+    margin-top: 1.5rem;
+    background: var(--gray-bg-color);
+}
+
+.login-container h3{
+    padding-top: 2rem;
+    font-size: 1.8rem;
+}
+
+.login-card{
+    display: flex;
+    flex-direction: column;
+    justify-content:center;
+    align-items: center;
+    padding: 1rem 3rem 2rem 3rem;
+    margin: 1rem 0 3rem 0;
+    border-radius: 10px;
```

Comment:
You can add these border radius values to a css variable for uniformity across the app (--sm, --md,--full-round)